### PR TITLE
Add Docker file

### PR DESCRIPTION
**Pull request by katopz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,6 +144,19 @@ Some additional things to try:
     A modern-ish C toolchain is required to compile `ggml`. A C++ toolchain
     should not be necessary.
 
+### Docker
+
+```shell
+# To build (This will take sometime, go grab some coffee).
```
Comment (philpax) [inline README.md:150]: `.` -> `:` at the end, `sometime` -> `some time`

File: README.md [hunk 2]
```diff
@@ -147,6 +144,19 @@ Some additional things to try:
...
+docker build -t llama-rs .
+
+# To run with prompt.
```
Comment (philpax) [inline README.md:153]: `.` -> `:` at the end

File: README.md [hunk 3]
```diff
@@ -147,6 +144,19 @@ Some additional things to try:
...
+docker run --rm --name llama-rs -it -v ${PWD}/data:/data -v ${PWD}/examples:/examples llama-rs -m data/gpt4all-lora-quantized-ggml.bin -p "Tell me how cool the Rust programming language is:"
+
+# To run with prompt file and repl (will wait for user input).
```
Comment (philpax) [inline README.md:156]: `.` -> `:` at the end

**Review by philpax — APPROVED** [review]
LGTM assuming it all works as expected and you can fix those minor typographic nits of mine.

**Comment by katopz** [comment]
> Also, are you OK to license this change and others under MIT/Apache2? ( #92 )

I'm OK 👌

**Comment by philpax** [comment]
Excellent, thank you 🚀

**Pull request merged**